### PR TITLE
feat(LLM): web3hub [LIVE-13124]

### PR DESCRIPTION
**Pull request by Justkant**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - Discover section only with the FF active

### 📝 Description

Setup FF for switching between discover and web3hub
Added @react-navigation/native-stack to test a truly native navigation
Added @shopify/flash-list to test the performances
Initial navigation and screens
Setup the integration tests

### ❓ Context

- **JIRA or GitHub link**: [LIVE-13124] 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-13124]: https://ledgerhq.atlassian.net/browse/LIVE-13124?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ComradeAERGO — DISMISSED** [review]
LGTM

File: apps/ledger-live-mobile/src/newArch/features/Web3Hub/__integrations__/web3hub.integration.test.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable i18next/no-literal-string */
+import * as React from "react";
+import { screen } from "@testing-library/react-native";
+import { render } from "@tests/test-renderer";
+import { Web3HubTest } from "./shared";
+
+describe("Web3Hub integration test", () => {
```
Comment (KVNLS) [inline apps/ledger-live-mobile/src/newArch/features/Web3Hub/__integrations__/web3hub.integration.test.tsx:7]: ❤️

**Review by KVNLS — APPROVED** [review]

**Comment by Justkant** [comment]
@KVNLS Do you think the cache on the CI can be the issue for the podfile.lock ?
https://github.com/LedgerHQ/ledger-live/actions/runs/9776787696/job/26990794818

**Comment by KVNLS** [comment]
> @KVNLS Do you think the cache on the CI can be the issue for the podfile.lock ? https://github.com/LedgerHQ/ledger-live/actions/runs/9776787696/job/26990794818

You need to make sure that locally you use the correct Ruby & bundler version, then you need to clean your cocoapod cache, if you don't have any change it might come from the cache.
But I already had the issue where it came from my local configuration and I had to clean `~/.cocoapods` folder.
To be sure you can run the CI locally, https://ledgerhq.atlassian.net/wiki/spaces/WXP/pages/4658397217/Run+MacOS+Runner+locally+with+Tart

**Pull request merged**